services/mobileappbuilder/index.md

```diff
@@ -9,11 +9,15 @@ copyright:
 {:shortdesc: .shortdesc}
 {:screen:.screen}
 {:codeblock:.codeblock}
+{:note:.deprecated}
+
+**This service is deprecated:** All instances of this service are deprecated. Existing instances can be used until 11 August 2016. For more information, see the [deprecation announcement blog](https://developer.ibm.com/bluemix/2016/07/12/mobile-app-builder-now-part-bluemix-mobile-dashboard/){: new_window}.
+{:deprecated}
```

Comment:
Why is that do you know? (I have the same {:deprecated} unfortunately. 

And I'm looking at the {:shortdesc} usage in the Getting started template and wondering which ones need this space? 
Any help is much appreciated :-)